addons/airway/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -24,14 +24,34 @@ class ACE_Medical_Treatment_Actions {
         litter[] = {};
         icon = QPATHTOF(ui\larynx.paa);
     };
+    class RemoveLarynxtubus: Larynxtubus {
+        displayName = CSTRING(Cancel_Larynxtubus);
+        displayNameProgress = CSTRING(action_removing);
+        medicRequired = QGVAR(medLvl_Larynxtubus);
+        treatmentTime = QGVAR(Larynxtubus_time);
+        items[] = {};
+        condition = QUOTE(!([_patient] call ace_common_fnc_isAwake) && (missionNamespace getVariable [ARR_2(QQGVAR(enable),true)]) && _patient getVariable [ARR_2(QQGVAR(kingLTisSet), false)]);
+        callbackSuccess = QFUNC(treatmentAdvanced_RemoveLarynxtubus);
+    };
     class Guedeltubus: Larynxtubus {
         displayName = CSTRING(Guedeltubus_Display);
         medicRequired = QGVAR(medLvl_Guedeltubus);
         treatmentTime = QGVAR(Guedeltubus_time);
+        condition = QUOTE(!([_patient] call ace_common_fnc_isAwake) && (missionNamespace getVariable [ARR_2(QQGVAR(enable),true)]) && !(_patient getVariable [ARR_2(QQGVAR(recovery),false)]) && !(_patient getVariable [ARR_2(QQGVAR(guedeltubusisSet), false)]) && !(_patient getVariable [ARR_2(QQGVAR(kingLTisSet), false)]));
         items[] = {"kat_guedel"};
         icon = QPATHTOF(ui\guedel.paa);
         callbackSuccess = QFUNC(treatmentAdvanced_airway);
     };
+    class RemoveGuedeltubus: RemoveLarynxtubus {
+        displayName = CSTRING(Cancel_Guedeltubus);
+        displayNameProgress = CSTRING(action_removing);
+        medicRequired = QGVAR(medLvl_Guedeltubus);
+        treatmentTime = QGVAR(Guedeltubus_time);
+        items[] = {};
+        icon = QPATHTOF(ui\guedel.paa);
+        condition = QUOTE(!([_patient] call ace_common_fnc_isAwake) && (missionNamespace getVariable [ARR_2(QQGVAR(enable),true)]) && _patient getVariable [ARR_2(QQGVAR(guedeltubusisSet), false)]);
```

Comment:
`(missionNamespace getVariable [ARR_2(QQGVAR(enable),true)])`
change to
missionNamespace getVariable [ARR_2(QQGVAR(enable),true)]

Remove () around the variable